etc/prometheus.yml.ctmpl

```diff
@@ -1,8 +1,8 @@
 # my global config
 global:
   scrape_interval:     15s # By default, scrape targets every 15 seconds.
-  evaluation_interval: 15s # By default, scrape targets every 15 seconds.
-  # scrape_timeout is set to the global default (10s).
+  evaluation_interval: 13s #
```

Comment:
The examples show the `evaluation_interval` slightly shorter than the `scrape_interval`, so...